### PR TITLE
redirect to welcome from base and /multicloud

### PR DESCRIPTION
**Pull request by willkutler**
Signed-off-by: Will Kutler <wkutler@redhat.com>

needed now that header charts have been removed

### CODE REVIEW

File: server/controllers/index.js [hunk 1]
```diff
@@ -23,7 +23,10 @@ const express = require('express'),
 const status = require('./status'),
       ui = require('./ui')
 
-router.all(['/', '/status', '/livenessProbe', '/readinessProbe'], status)
+router.all(['/status', '/livenessProbe', '/readinessProbe'], status)
+router.all(['/', '/multicloud'], (req, res) => {
+    res.redirect('/multicloud/welcome')
+})
```
Comment (dhaiducek) [inline server/controllers/index.js:29]: This looks good. I'll double check with Cluster Lifecycle before we claim the `/multicloud` path just to make sure.

**Review by dhaiducek — COMMENTED** [review]
Would it be a good to add some quick tests to check the URL in the welcome page tests to make sure these redirects are working or is that overkill?

**Comment by willkutler** [comment]
> Would it be a good to add some quick tests to check the URL in the welcome page tests to make sure these redirects are working or is that overkill?

sure, i'll add one

**Review by dhaiducek — APPROVED** [review]
LGTM!

**Pull request merged**